config/application.rb

```diff
@@ -34,15 +34,6 @@
 
 module Openfoodnetwork
   class Application < Rails::Application
-    config.middleware.insert_before(
-      ActionDispatch::Cookies,
-      SessionCookieUpgrader, {
-        old_key: "_session_id",
-        new_key: "_ofn_session_id",
-        domain: ".#{ENV['SITE_URL'].gsub(/^(www\.)|^(app\.)|^(staging\.)|^(stg\.)/, '')}"
-      }
-    ) if Rails.env.staging? || Rails.env.production?
-
```

Comment:
Great. I'm assuming that you also want to delete lib/session_cookie_upgrader.rb. This is still in draft. I could move it back or maybe I quickly do it and ready it for review.